archunit/src/main/java/com/tngtech/archunit/library/plantuml/rules/PlantUmlArchCondition.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.tngtech.archunit.library.plantuml;
+package com.tngtech.archunit.library.plantuml.rules;
 
```

Comment:
1. Should `class PlantUmlArchCondition` itself be declared as `@PublicAPI`?
2. Is `..rules` really a good package name for a _condition_?

Reply:
1. Honestly, I never established a clear convention for this :thinking: It implicitly is, because it has `@PublicAPI` members, but maybe it would be better from a user's perspective to make the class itself also `@PublicAPI` :thinking: But then it should maybe be done for all classes, because I doubt it's the only candidate :man_shrugging: 
2. My intention was to create one bucket for everything related to creating rules from PlantUML. I didn't really want to orient the package name so much on what we have right now (right now we only have a condition), but more on the vision for this package (everything about PlantUML rules). My vision is to put something in there to also create PlantUML diagrams, and then the distinction between "diagram_creation"/"rendering"(whatever we'll call it) and "rules" would make sense to me, as a rough splitting into these two different semantic parts.